scania-digital-design-system/sdds-website
# Improvement/table colour and size

**Describe pull-request**  
- Fix table typography styling
- Added table from CMS, with sizing options

**Solving issue**  

Fixes: #150 

**How to test**  
- Test together with CMS admin https://github.com/scania-digital-design-system/sdds-cms/pull/29
- Add table component in your local, or use content.json that I added on this PR
- Change fetchData to localhost:1339, or you can use content.json I attached as the example
- Added example on content.json, check Components > Text Field > Style > States > check table Default & Hover, compare other table below
